ingest-app/src/cmr/ingest/api/core.clj

```diff
@@ -233,6 +231,16 @@
   [concept]
   (pr-str (dissoc concept :metadata)))
 
+(defn log-concept-with-metadata-size
```

Comment:
I think this should be moved out of here and into cmr.ingest.services.helper

Reply:
the cmr.ingest.api.core namespace is starting to look really cluttered.

Reply:
Reading through the namespace it looks like it was always a dumpster for helper functions.  I think we can probably leave this here then.

Reply:
I have no problem of moving it. But I want to understand what core is used for because other common functions were in there too, so we need to be consistent...